src/Cxsearch/Search.php

```diff
@@ -130,8 +130,11 @@ public function orQuery($target, $value=null, $boost=null)
 
     private function _filter($target, $operator, $value, $prefix=null)
     {
-        if (!is_numeric($value)) {
+
+        if (!is_numeric($value)&&!is_array($value)) {
             $value = '"' . $value . '"';
+        } else if(is_array($value)){
+            $value = 'range('.implode(',', $value).')';
```

Comment:
We'd like to move support for ranges into a public `filterRange` method to handle the value appropriately.

Also add a flag to filter method to control quoting since not all values need to be quoted.